remote_error_monitor.c

```diff
@@ -148,8 +158,17 @@ static void remote_error_monitor_error_callback(int type, const char *error_file
   //rem_append_backtrace(&trace_str);
   smart_str_0(&trace_str);
   remote_error_monitor_process(REMOTE_ERROR_MONITOR_ERROR, error_filename, error_lineno, args, trace_str.s ? ZSTR_VAL(trace_str.s) : "");
-}
 
+  // Gentle roll-out: first, do not call the old error handler at all in the live environment.
+  // Once we are ready to enable this on dev / multidev environments, we can change
+  // this check to only call through to the old_error_cb on the live environment
+  // if it was deployed AFTER a given date (and always call through on dev/multidev).
```

Comment:
We probably aren't going to do that after-a-given-date thing. @tylerdigital favored the "band-aid off quickly" solution instead of taking on the extra work. Comment is okay, though, as it will be short-lived.